packages/app-desktop/commands/toggleExternalEditing.ts

```diff
@@ -23,7 +22,7 @@ export const runtime = (): CommandRuntime => {
 				void CommandService.instance().execute('startExternalEditing', noteId);
 			}
 		},
-		enabledCondition: enabledCondition(declaration.name),
+		enabledCondition: 'oneNoteSelected && !noteIsReadOnly',
```

Comment:
Shouldn't we have "`(!modalDialogVisible || gotoAnythingVisible)" (from enabledCondition) too?

Reply:
Before 77482a0c956987434a4e1f7d645aeec8d51292eb, it was just `oneNoteSelected`. I can add the additional conditions, though!